### PR TITLE
Develop upstream sync 230619

### PR DESCRIPTION
**Pull request by draganmladjenovic**
https://github.com/ROCmSoftwarePlatform/frameworks-internal/issues/4665

### CODE REVIEW

**Review by i-chaochen — CHANGES_REQUESTED** [review]
@draganmladjenovic  Good fix! please upstream this commit 👍

**Review by i-chaochen — CHANGES_REQUESTED** [review]
@draganmladjenovic  Good fix! please upstream this commit as well 👍

**Review by i-chaochen — CHANGES_REQUESTED** [review]
@draganmladjenovic  Good fix! 👍  please upstream this commit as well.

**Comment by draganmladjenovic** [comment]
retest Ubuntu-CPU please

**Comment by draganmladjenovic** [comment]
retest Ubuntu-CPU please

**Comment by draganmladjenovic** [comment]
retest Ubuntu-GPU-single please

**Comment by draganmladjenovic** [comment]
retest Ubuntu-GPU-single please

**Comment by draganmladjenovic** [comment]
retest Ubuntu-GPU-single please

**Comment by draganmladjenovic** [comment]
retest Ubuntu-GPU-single please

**Comment by i-chaochen** [comment]
retest Ubuntu-CPU please

**Comment by draganmladjenovic** [comment]
retest Ubuntu-GPU-single please

**Comment by draganmladjenovic** [comment]
retest Ubuntu-GPU-single please

**Comment by draganmladjenovic** [comment]
retest Ubuntu-GPU-single please

File: tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc
Comment (i-chaochen) [inline tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc:1]: This is for fixing PR https://github.com/tensorflow/tensorflow/commit/992a68adde1810756f2930f57e6108f891fa7e16

**Comment by draganmladjenovic** [comment]
retest Ubuntu-GPU-single please

**Comment by draganmladjenovic** [comment]
retest Ubuntu-GPU-single please

**Review by i-chaochen — APPROVED** [review]

**Pull request merged**